src/Combiner.cpp

```diff
@@ -422,6 +422,9 @@ void CombinerInfo::_saveShadersStorage() const
 
 	u32 totalWritten = 0;
 	std::vector<char> allShaderData;
+	
+	//Clear the GL error flag so that follow up check don't falsely return errors.
+	isGLError();
```

Comment:
@gonetz With hardware lighting enabled, it's this causing the error flag:

```
void UniformSet::_updateLightUniforms(UniformSetLocation & _location, bool _bForce)
{
	for (s32 i = 0; i <= gSP.numLights; ++i) {
		_location.uLightDirection[i].set(&gSP.lights[i].x, _bForce);
		_location.uLightColor[i].set(&gSP.lights[i].r, _bForce);
	}
}
```

I did not try to dig to see what is causing the error flag hardware lighting is disabled.